docs/neo-n3/dapp_development/advanced.md

```diff
@@ -1,17 +1,9 @@
 ## Iterators
 
-**TL;DR** You can use `SmartContract.callFunctionReturningIterator(...)` to invoke a method that returns an iterator and
-traverse through the iterator using the resulting `Iterator` object and its utility methods.
+**TL;DR:** You can use `SmartContract.callFunctionReturningIterator(...)` to invoke a method that returns an iterator and traverse through the iterator using the resulting `Iterator` object and its utility methods.
```

Comment:
It seems that this was not modified at all. Can you please return to its original format?

We can fix the word wrap in another PR, if necessary.